packages/devtools_app/lib/src/framework/notifications_view.dart

```diff
@@ -139,7 +139,21 @@ class _NotificationsState extends State<_Notifications> with AutoDisposeMixin {
     });
   }
 
-  Widget _buildOverlay(BuildContext context) {
+  @override
+  Widget build(BuildContext context) {
+    return widget.child;
+  }
+}
+
+class _NotificationOverlay extends StatelessWidget {
```

Comment:
@kenzieschmoll I moved some but not all cases where we had functions that should have been widgets over to using widgets.